examples/distance.lua

```diff
@@ -0,0 +1,56 @@
+-- @example GPM Implementation strategy 'distance' and creating map.
+-- Create a map based on the endpoints, and the cells.
+
+-- import gpm
+import("gpm")
+
+-- create the CellularSpace
+local csCenterspt = CellularSpace{
+	file = filePath("communities.shp", "gpm"),
+	geometry = true
+}
+
+local csLine = CellularSpace{
+	file = filePath("roads.shp", "gpm"),
+	geometry = true
+}
+
+local farms = CellularSpace{
+	file = filePath("farms_cells.shp", "gpm"),
+	geometry = true
+}
+
+-- create a Network with the distance of the end points to routes
+local network = Network{
+	target = csCenterspt,
+	lines = csLine,
+	weight = function(distance, cell)
+		if cell.STATUS == "paved" then
+			return distance / 5
+		else
+			return distance / 2
+		end
+	end,
+	outside = function(distance, cell) return distance * 2 end
+}
+
+-- creating a GPM with the distance of the entry points for the routes
+local gpm = GPM{
+	network = network,
```

Comment:
The network here is unnecessary. It should be replaced by an argument called `destination`.